apps/api/v2/src/ee/event-types/event-types_2024_06_14/constants/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const DEFAULT_EVENT_TYPES = {
+  thirtyMinutes: { length: 30, slug: "thirty-minutes", title: "30 Minutes" },
+  thirtyMinutesVideo: {
+    length: 30,
+    slug: "thirty-minutes-video",
+    title: "30 Minutes",
+    locations: [{ type: "integrations:daily" }],
+  },
+  sixtyMinutes: { length: 60, slug: "sixty-minutes", title: "60 Minutes" },
+  sixtyMinutesVideo: {
+    length: 60,
+    slug: "sixty-minutes-video",
+    title: "60 Minutes",
+    locations: [{ type: "integrations:daily" }],
+  },
+};
```

Comment:
is this supposed to be the same values than in apps/api/v2/src/ee/event-types/event-types_2024_04_15/constants/constants.ts

Reply:
Yes. DEFAULT_EVENT_TYPES are already in format compatible with Prisma. I could store them in format that the new event-types version expects POST event-type body, but then I need to convert them into this format anyway.